optimize/client/src/components/Analysis/TaskAnalysis/OutlierDetailsModal.tsx

```diff
@@ -39,17 +40,19 @@ export default function OutlierDetailsModal({
     <Modal open onClose={onClose} className="OutlierDetailsModal" size="lg">
       <Modal.Header title={t('analysis.task.detailsModal.title', {name})} />
       <Modal.Content>
-        <p className="description">
-          {t('analysis.task.totalFlowNodeInstances', {count: totalCount})}
-          <span>{getOutlierSummary(count, relation)}</span>
-        </p>
-        <h2>{t('analysis.task.detailsModal.durationChart')}</h2>
+        <Stack gap={4}>
+          <p className="description">
+            {t('analysis.task.totalFlowNodeInstances', {count: totalCount})}
+            <span>{getOutlierSummary(count, relation)}</span>
+          </p>
+          <h4 className="subTitle">{t('analysis.task.detailsModal.durationChart')}</h4>
```

Comment:
Is this class `subTitle` a global one? I don't see it on the associated `.scss` file.